src/entity/ChatMessage.java

```diff
@@ -0,0 +1,12 @@
+package entity;
+
+import java.util.List;
+
+public interface ChatMessage {
+    public String getTime();
+    public String getMessage();
+    public List<User> getUsers();
```

Comment:
It's good to have a list of `User`s here, but you should add a `uid` attribute to the `User` class if you want to be able to use this properly.

Reply:
removed list of users, made it purely about roomID.